core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -609,27 +610,32 @@ protected function doReplace($strBuffer, $blnCache)
 				case 'iflng':
 					if ($elements[1])
 					{
+						$pageLanguage = LocaleUtil::formatAsLocale($objPage->language);
 						$langs = StringUtil::trimsplit(',', $elements[1]);
 
 						// Check if there are wildcards (see #8313)
 						foreach ($langs as $k=>$v)
 						{
-							if (substr($v, -1) == '*')
+							if (substr($v, -1) === '*')
 							{
-								$langs[$k] = substr($v, 0, -1);
+								$langs[$k] = LocaleUtil::formatAsLocale(substr($v, 0, -1));
```

Comment:
not really sure what this code does?

Reply:
It converts the language ID `en_US` into the locale `en-US` (or vice versa).

Reply:
I think it removes the trailing wildcard character `*`.

Reply:
Agree, if the insert tag is used like `{{iflng::de*}}`, see https://docs.contao.org/manual/de/artikelverwaltung/insert-tags/#verschiedenes

Reply:
Glad I've documented this not too long ago 😎